sysutils/rawhide/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           makefile 1.0
+PortGroup           legacysupport 1.0
+
+name                rawhide
+version             3.2
```

Comment:
Since this is a Github project, `name` and `version` do not need to be specified, and can be completely replaced with `github.setup`:

```
github.setup        raforg rawhide 3.2 v
```

Reply:
Thanks. I was aware of that, but I wanted it this way, so that the name and version appear (almost) first in the file, followed by the rest of the package information, followed by the fetch details. I didn't like the idea that the name and version, which are the primary identifiers of the package, wouldn't appear until the middle of the Portfile. I decided that the unnoticeable wastefulness of github.setup setting name and version to their existing values was worthwhile to improve the readability (in my opinion) of the Portfile. I hope that's not a problem. And before you suggest it, I didn't want to put the github/fetch section before the package information section either. I like the order of sections the way I have it: groups, id, pkg details, fetch, configure+make, test, livecheck. Although, maybe the id should appear first.

Reply:
Thanks. Done. See above.

Reply:
Makes sense.